testnets/osmosistestnet/chain.json

```diff
@@ -0,0 +1,106 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "osmosistestnet",
+    "status": "live",
+    "network_type": "testnet",
+    "pretty_name": "Osmosis Testnet",
+    "chain_id": "osmo-test-4",
+    "bech32_prefix": "osmo",
+    "daemon_name": "osmosisd",
+    "node_home": "$HOME/.osmosisd",
+    "genesis": {
+        "genesis_url": "https://github.com/osmosis-labs/networks/raw/main/osmo-test-4/genesis.tar.bz2"
+    },
+    "key_algos": [
+        "secp256k1"
+    ],
+    "slip44": 118,
+    "fees": {
+        "fee_tokens": [
+            {
+                "denom": "uosmo",
+                "fixed_min_gas_price": 0,
+                "low_gas_price": 0,
+                "average_gas_price": 0.025,
+                "high_gas_price": 0.04
+            }
+        ]
+    },
+    "staking": {
+        "staking_tokens": [
+            {
+                "denom": "uosmo"
+            }
+        ]
+    },
+    "codebase": {
+        "git_repo": "https://github.com/osmosis-labs/osmosis",
+        "recommended_version": "v7.0.2",
+        "compatible_versions": [
+            "v7.0.2"
+        ],

```

Comment:
I don't think this is right.  I'm pretty sure osmosis testnet is on v11

Reply:
Correct, osmo-test-4 is on v11 currently